frontend/src/metabase/query_builder/components/CompareAggregations/utils.ts

```diff
@@ -93,3 +90,29 @@ export const canSubmit = (
   const areColumnsValid = columns.length > 0;
   return isPeriodValid && areColumnsValid;
 };
+
+export function canAddTemporalCompareAggregation(
+  query: Lib.Query,
+  stageIndex: number,
+  aggregations?: Lib.AggregationClause[],
```

Comment:
Is there any case where `aggregations` differ from `Lib.aggregations(query, stageIndex)`? If yes this would be weird. If not let's remove this parameter and always call `Lib.aggregations`. The call is memoized so there is no performance cost